core/kazoo_documents/src/kzd_freeswitch.erl

```diff
@@ -52,11 +52,14 @@
 
 -include("kz_documents.hrl").
 
+-type data() :: kz_proplist().
```

Comment:
I thought about that too;.most of the `kazoo_document` modules manipulate JSON docs from Couch; this module handles FreeSWITCH proplists. We could move this to ecallmgr since its specific to that application (and then we could leverage this .hrl file to maybe auto-generate this module too). Perhaps I'll file a ticket for that as a future improvement.